### DISCUSSION TITLE
After this new update, all videos downloaded as black screen no audio

### DISCUSSION THREAD

**Discussion by eziosavva** [aclap-dev/video-downloadhelper#146] [Help and Q&A]
### Your question

With the new update, I think I downloaded all the files, but something went wrong. The video I downloaded from Udemy appears black, and when I tried playing other videos, they didn't work.

### Addon & CoApp version and details

version: 8.2.0.24
channel: stable
build date: 2024-02-16
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-GB
license: accepted
key (only 16 first characters): e884e31f53334186
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.17",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac13",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/Users/eziosavva",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.1.1-tessus"
}

**Comment by GrampaWildWilly** [comment]
We've investigated this & a fellow user determined that for any given course, the first lesson is perfectly downloadable by VDH but the second & subsequent lessons of any course are protected by DRM.  VDH has never, does not now, & will never download content protected by DRM.  What you did is proof of that.  The content is protected by DRM.  VDH does not download such content.  You'll need to use a screen recorder for those lessons.  Too bad.

**Comment by paulrouget** [comment]
I need a link to the Udemy page.
